toast.yml

```diff
@@ -5,7 +5,6 @@ tasks:
       - pkg/webview/view.proto
     output_paths:
       - pkg/webview/view.pb.go
-      - pkg/webview/view.pb.gw.go
```

Comment:
???

Reply:
We don't need this anymore, we don't use the grpc services in Tilt.

Reply:
I guess I should delete the generated code too, then. Will do.